bears/php/PHPMessDetectorBear.py

```diff
@@ -0,0 +1,38 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import \
+    DistributionRequirement
+
+
+@linter(executable='phpmd',
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>\d+)(?P<message>.*)')
+class PHPMessDetectorBear:
+    """
+    The bear takes a given PHP source code base and looks for several
+    potential problems within that source. These problems can be things like:
+
+    - Possible bugs
+    - Suboptimal code
+    - Overcomplicated expressions
+    - Unused parameters, methods, properties
+    """
+
+    LANGUAGES = {'PHP'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='phpmd')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Complexity', 'Unused Code', 'Redundancy',
+                  'Variable Misuse'}
+    SEE_MORE = 'https://phpmd.org/about.html'
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         phpmd_ruleset: list):
```

Comment:
The ruleset is a mandatory argument, so `phpmd s.php text` yields nothing whereas `phpmd s.php text cleancode` yields results.